_ingest-pipelines/processors/set.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: default
+title: Set
+parent: Ingest processors
+nav_order: 240
+---
+
+# Set processor
+
+The `set` processor adds or updates fields in a document. It sets one field and associates it with the specified value. If the field already exists, its value is replaced with the provided one.
```

Comment:
```suggestion
The `set` processor adds or updates fields in a document. It sets one field and associates it with the specified value. If the field already exists, its value is replaced with the provided one, unless the override parameter is set to false. When override is false and the specified field exists, the value of the field will remain unchanged.
```

Reply:
@gaobinlong Please confirm the change accurately reflects your comment. Thank you.

Reply:
Edited suggestion